amqp/1.8/modules/ROOT/pages/amqp-documentation.adoc

```diff
@@ -0,0 +1,802 @@
+| Consumer Tag a| String |  The consumer tag to use for the consumer involved in the operation |  | 
+| Maximum Wait a| Number |  Maximum time to wait for a message to arrive before timeout |  10000 | 
+| Maximum Wait Unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  Time unit to use in the maximumWaitTime configuration |  MILLISECONDS | 
+| Create Fallback Queue a| Boolean |  |  | 
+| Transactional Action a| Enumeration, one of:
+
+** ALWAYS_JOIN
+** JOIN_IF_POSSIBLE
+** NOT_SUPPORTED |  information of the current transaction in case it exists |  JOIN_IF_POSSIBLE | 
+| Streaming Strategy a| * <<repeatable-in-memory-stream>>
+* <<repeatable-file-store-stream>>
+* non-repeatable-stream |  Configures how Mule processes streams. Repeatable streams are the default behavior. |  | 
+| Target Variable a| String |  Name of the variable that storesoperation's output will be placed |  | 
+| Target Value a| String |  An expression to evaluate against the operation's output and store the expression outcome in the target variable |  `#[payload]` | 
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> |  Retry strategy in case of connectivity errors. |  | 
+|===
+
+==== Output
+[%autowidth.spread]
+|===
+|Type |Any
+| Attributes Type a| <<AmqpAttributes>>
+|===
+
+=== For Configurations
+* <<config>> 
+
+==== Throws
+* AMQP:QUEUE_NOT_FOUND 
+* AMQP:CONNECTIVITY 
+* AMQP:CONSUMING 
+* AMQP:NO_CHANNEL_AVAILABLE 
+* AMQP:RETRY_EXHAUSTED 
+* AMQP:CREATION_NOT_ALLOWED 
+* AMQP:TIMEOUT 
+
+
+[[publish]]
+=== Publish
+`<amqp:publish>`
+
+
+Enables you to publish a single AMQP message to a given exchange.
+
+
+==== Parameters
+[%header,cols="20s,20a,35a,20a,5a"]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | Name of the configuration to use. | | x
+| Exchange Name a| String |  The name of the exchange to publish the message to |   | 
+| Routing Keys a| Array of String |  List of routing keys |  | 
+| Delivery Mode a| Enumeration, one of:
+
+** TRANSIENT
+** PERSISTENT |  The delivery mode to use when publishing to the AMQP broker |  | 
+| Transactional Action a| Enumeration, one of:
+
+** ALWAYS_JOIN
+** JOIN_IF_POSSIBLE
+** NOT_SUPPORTED |  information of the current transaction in case it exists |  JOIN_IF_POSSIBLE | 
+| Send Correlation Id a| Enumeration, one of:
+
+** AUTO
+** ALWAYS
+** NEVER |  options on whether to include an outbound correlation ID or not |  | 
+| Create Fallback Exchange a| Boolean |  |  | 
+| Skip Exchange Validation a| Enumeration, one of:
+
+** YES |  Skips the exchange validation. By default, the Publish operation validates the exchange if you do not configure the Skip Exchange Validation field. While skipping validation can enhance performance, this approach also risks message loss if the exchange does not exist, depending on the broker's configuration. |  | 
```

Comment:
Suggestion: 
** YES | Skips the Exchange validation +
The Publish operation validates the exchange if you do not configure the Skip Exchange Validation field +
While skipping validation can enhance performance, this approach also risks message loss if the exchange does not exist, depending on the broker's configuration.